in_toto/runlib.py

```diff
@@ -350,15 +350,29 @@ def execute_link(link_cmd_args, record_streams):
     - The return value of the executed command.
   """
   if record_streams:
-    return_code, stdout_str, stderr_str = \
-        securesystemslib.process.run_duplicate_streams(link_cmd_args)
+    if (quiet == False): #record_streams true, quiet false
+      return_code, stdout_str, stderr_str = \
+      securesystemslib.process.run_duplicate_streams(link_cmd_args)
+    else: #record_streams true, quiet true
+     process = securesystemslib.process.run(link_cmd_args, check=False,
```

Comment:
Also, these have been indented with a single space instead of two.

Reply:
I think I made the changes you suggested! Let me know if it hasn't correctly been updated -- and thanks so much!